Basic Comcepts/4/file_info.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+print 'Please enter path of file: '
+path = gets.chomp
+
+puts "File name: #{File.basename(path)}"
+puts "Base name: #{File.basename(path, '.*')}"
+puts "Extension: #{File.extname(path)}"
+puts "Path name: #{File.expand_path(path)}"
```

Comment:
We need to only show directory path
```suggestion
puts "Path name: #{File.dirname(path)}"
```